docs/decisions/0006-transmit-updates-only.rst

```diff
@@ -0,0 +1,87 @@
+Transmitting Only Updates IDA
+-----------------------------
+
+Status
+======
+
+Draft
+
+Context
+=======
+The transmit_content_metadata jenkins job (along with the underlying integrated_channels.transmit_content_metadata()
+task and export() method) makes an uncomfortably large number of calls to the the enterprise-catalog service’s
+/api/v1/enterprise-catalog/<uuid>/get_content_metadata endpoint with a relatively low number of customers. The resulting
+response payloads are usually quite large which results in the service’s shared Aurora DB often being pinned at 100% CPU
```

Comment:
Simplify for OS community: "...which results in unacceptably high database load"